src/hypertrace/agent/__init__.py

```diff
@@ -172,11 +172,11 @@ def register_processor(self, processor): # pylint: disable=R1710
         if self.is_enabled():
             return self._init.register_processor(processor)
 
-    def get_config(self):
-        '''Return configuration object'''
-        return self._config
+#    def get_config(self) -> r:
+#        '''Return configuration object'''
+#        return self._config.get_config()
 
-    def is_enabled(self): # pylint: disable=R0201
+    def is_enabled(self) -> bool: # pylint: disable=R0201
```

Comment:
+1 I think we should include return type as much as possible

Reply:
Done.